packages/react-components/react-toast/src/components/Toaster/useFocusManagement.ts

```diff
@@ -0,0 +1,70 @@
+import { isHTMLElement } from '@fluentui/react-utilities';
+import * as React from 'react';
+import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
+import { ArrowDown, ArrowUp } from '@fluentui/keyboard-keys';
+import { toastContainerClassNames } from '../ToastContainer';
+
+/**
+ * @internal
+ * Toasts  should be arrow navigable and focus should be trapped in a stack of tasts
+ * This is a temporary measure, Tabster does not have an API yet to enable mover arrow keys from within grouppers
+ * Once tabster fully supports this use case, remove this hook
+ */
+export function useFocusManagement_unstable() {
+  const { targetDocument } = useFluent();
+  return React.useCallback(
+    (el: HTMLDivElement | null) => {
+      if (!el || !targetDocument) {
+        return;
+      }
+
+      const toastContainerWalker = targetDocument.createTreeWalker(el, NodeFilter.SHOW_ELEMENT, {
+        acceptNode(node: Node) {
+          if (isHTMLElement(node) && node.classList.contains(toastContainerClassNames.root)) {
+            return NodeFilter.FILTER_ACCEPT;
+          }
+
+          return NodeFilter.FILTER_SKIP;
+        },
+      });
+
+      el.addEventListener('keydown', e => {
```

Comment:
I don't see a matching `.removeListener()` call. Can we clean up it properly?

Reply:
Done in https://github.com/microsoft/fluentui/pull/28660/commits/4d12d8dbe0e2e7b617af8fe810f94df08c8a3896